### PR TITLE
chore(makefile): Add convenience tasks for creating, deleting, and updating k8s resources

### PR DESCRIPTION
**Pull request by jchauncey**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -73,6 +77,22 @@ kube-clean:
 	kubectl delete rc deis-${SHORT_NAME}
 	kubectl delete svc deis-${SHORT_NAME}
 
+kube-delete:
```
Comment (arschles) [inline Makefile]: `kube-delete` and `kube-create` seem to have a lot of overlapping functionality with `kube-clean`, `kube-rc`/`kube-service` (respectively). Can you delete one (don't have any preference which) or merge them?

Comment (jchauncey) [inline Makefile]: yeah I was going to delete them. I didnt know if you guys have specific scripts using those or not. I just know we have kube-create/delete in workflow and other repos and its pretty handy. I'll update the PR in just a sec

Comment (arschles) [inline Makefile]: >I didnt know if you guys have specific scripts using those or not

Nope, don't think so

>I just know we have kube-create/delete in workflow and other repos and its pretty handy

:+1: 

>I'll update the PR in just a sec

Ok, cool. LGTM

File: Makefile [hunk 2]
```diff
@@ -58,20 +62,23 @@ docker-push:
...
 
-kube-clean:
-	kubectl delete rc deis-${SHORT_NAME}
-	kubectl delete svc deis-${SHORT_NAME}
+kube-update: update-manifests
+	kubectl delete -f manifests/deis-builder-rc.tmp.yml
+	kubectl create -f manifests/deis-builder-rc.tmp.yml
+
+update-manifests:
+	sed 's#\(image:\) .*#\1 $(IMAGE)#' manifests/deis-builder-rc.yml \
```
Comment (aledbf) [inline Makefile]: The extension of the files is `yaml` not `yml`

**Comment by jchauncey** [comment]
closing this PR per discussions from last week

**Pull request closed without merging**